sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_region.c

```diff
@@ -96,6 +96,10 @@ uint32_t gnrc_lorawan_pick_channel(gnrc_lorawan_t *mac)
 
 void gnrc_lorawan_process_cflist(gnrc_lorawan_t *mac, uint8_t *cflist)
 {
+#ifdef GNRC_LORAWAN_SINGLE_CHANNEL
+    (void)mac;
+    (void)cflist;
+#else
```

Comment:
```suggestion
if(!IS_USED(GNRC_LORAWAN_SINGLE_CHANNEL) {
```

Reply:
Done

Reply:
I don't think this can be done. As I understand, @gschorcht proposes that `GNRC_LORAWAN_SINGLE_CHANNEL` holds the single frequency value, so it would not be a boolean switch.

Reply:
Yes, `GNRC_LORAWAN_SINGLE_CHANNEL` defines a value. `IS_USED' and `IS_ACTIVE' test for a macro that is 1, 0 or not defined. So they are not applicable here.

Reply:
ah, misread and though it was a boolean